fala/templates/adviser/accessibility_statement.html

```diff
@@ -0,0 +1,117 @@
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>change colours, contrast levels and fonts using browser or device settings</li>
+      <li>zoom in up to 400% without the text spilling off the screen</li>
+      <li>navigate most of the website using a keyboard or speech recognition software</li>
+      <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+    </ul>
+
+    <p class="govuk-body">
+      We’ve also made the website text as simple as possible to understand.
+    </p>
+
+    <p class="govuk-body">
+      <a class="govuk-link" href="https://mcmw.abilitynet.org.uk/" target="_blank" rel="noopener external noreferrer">AbilityNet</a> has
+      advice on making your device easier to use if you have a disability.
+    </p>
+
+    <h2 class="govuk-heading-m">How accessible this website is</h2>
+
+    <p class="govuk-body">
+      We know some parts of this website are not fully accessible:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>elements of the results page do not meet minimum contrast standards</li>
+      <li>Google Translate does not translate technical terms into Plain English</li>
+      <li>hyperlinks to google maps are not specific about the listing they are related to</li>
+    </ul>
+   
+    <h2 class="govuk-heading-m">Feedback and contact information</h2>
+
+    <p class="govuk-body">
+      We’re always looking to improve the accessibility of this website. 
+      If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact us by email at 
+      <a class="govuk-link email" href="mailto:eligibility@justice.gov.uk">eligibility@justice.gov.uk</a>.
+    </p>
+
+    <h2 class="govuk-heading-m">Enforcement procedure</h2>
+
+    <p class="govuk-body">
+      The Equality and Human Rights Commission (<abbr>EHRC</abbr>) is responsible for enforcing the Public
+      Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations
+      2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to
+      your complaint, <a class="govuk-link" href="https://www.equalityadvisoryservice.com/" target="_blank" rel="noopener external noreferrer">contact the Equality
+      Advisory and Support Service (<abbr>EASS</abbr>)</a>.
+    </p>
+
+    <h2 class="govuk-heading-m">
+      Technical information about this website’s accessibility
+    </h2>
+
+    <p class="govuk-body">
+      The Legal Aid Agency is committed to making its website accessible, in accordance with the
+      Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations
+      2018.
+    </p>
+
+    <h2 class="govuk-heading-m">Compliance status</h2>
+
+    <p class="govuk-body">
+      This website is <strong>partially compliant</strong> with the Web Content Accessibility Guidelines version 2.2 
```

Comment:
Question - one for the content/copywriters in the team. 

Should this read as "Web Content Accessibility Guidelines (WCAG)" - as we use the acronym  'WCAG' further down the page?